README.md

```diff
@@ -176,6 +178,43 @@ Find `.gov` certificates in the first 2 pages of Censys API results, waiting 5 s
 ./gather censys --suffix=.gov --start=1 --end=2 --delay=5
 ```
 
+### a11y setup
+
+Because scanning 1,000+ domains with `pa11y` takes a prohibitively long time, we're relying on [AWS Lambda](https://aws.amazon.com/lambda/) to provide parallelization.
+
+This requires:
+
+1) An AWS account with access to Lambda
+2) A `pa11y-lambda` function (follow the instructions [here](https://github.com/18F/pa11y-lambda)).
+
+Once those are set up, copy the `.env.example` file, rename it `.env` and fill in the following values:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_REGION_NAME` (`us-east-1` should work fine)
+- `AWS_LAMBDA_PA11Y_FUNCTION_NAME` (whatever you ended up naming the Lambda function)
+
+<hr />
+A brief note on redirects:
+
+For the accessibility scans we're running at 18F, we're using the `inspect` scanner to follow redirects _before_ the accessibility scan runs. For example, if aaa.gov redirects to bbb.gov, `pa11y` will run against bbb.gov.
```

Comment:
However, I believe this will produce a result _for_ `aaa.gov`, right? So you'd get `cache/pa11y/aaa.gov.json`, even if the scan was run against `bbb.gov`. If this isn't the case, you could get inconsistent results and some scans overwriting others.